model/github_installation.rb

```diff
@@ -14,4 +14,8 @@ def installation_url
     end
     "https://github.com/settings/installations/#{installation_id}"
   end
+
+  def total_active_runner_cores
+    runners.sum(&:used_vm_cores)
```

Comment:
This line sends N+1 queries, where N is the number of requested runners. For example, if a customer requests 300 runners, each one will send 300 requests, resulting in a total of 90,000 requests. To reduce this, you can use either eager loading, which requires 2 queries, or a join operation, which requires just 1 query, to calculate the sum.

```suggestion
    runners.eager(:vm).all.sum(&:used_vm_cores)
```

I didn't check syntax for the join version
```suggestion
    runners_dataset.left_join(:vm, id: :vm_id).sum(Sequel[:vm][:cores])
```